### ISSUE TITLE
Add License

### ISSUE THREAD

**Issue by gabzim** [arg20/circle-to-radius#18]
We should pick a license and add it to the project files

**Comment by johachi** [comment]
Haven't you already picked ISC? Or did you mean to all files in the project?

**Comment by gabzim** [comment]
Yes I meant in the project files, is that necessary? I see a lot of open source projects doing it and figured you might need to add it to every file.

**Comment by johachi** [comment]
I googled a bit. It seems like we should have a `LICENSE.txt` or a `LICENSE.md` in the root of the project, and then also a disclaimer in every source file.

I think the disclaimer (for each source file) could easily be written when linting the file.

**Comment by johachi** [comment]
@gabzim I added a license file and a section to the readme in this PR: #20 

Could you please see if you can add license info automatically in the source file on linting?

TODO:
- [x] Add `license.txt`
- [x] Add license info to `readme.md`
- [ ] Add license disclaimer to source files

**Comment by johachi** [comment]
I see now that you assigned yourself to this issue. Hope you don't mind that I made a small PR addressing some points on this PR. Feel free to reject it if it's not what you imagined :)
